gilded-rose-llc-frontend/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
```

Comment:
Suggestion: You might want to add `baseUrl` or `paths` for importing components later on. More details here: https://nextjs.org/docs/advanced-features/module-path-aliases

Reply:
Will def come in handy thank you! I added the base URL path of "." in [a3e2cf3](https://github.com/jramnani/GildedRoseJavaLLC/pull/27/commits/a3e2cf346e22664093ca07f29780f1c70fcac0ef)

@jonmanahan let's keep this in mind for when we add a components directory and can use `"paths"`

Reply:
👍